stingray/modeling/parameterestimation.py

```diff
@@ -1284,10 +1284,14 @@ def sample(self, lpost, t0, cov=None,
         """
         self.lpost = lpost
 
-        fit_res = ParameterEstimation.fit(self, self.lpost, t0, neg=True)
+        if cov is None:   
```

Comment:
Is this meant to be in this PR? If it's a small and quick fix we can include it, but just checking that you are aware of its presence here.

Reply:
Yes, it's a really small fix of something that's been annoying me for a long time.